apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -8,26 +8,21 @@ import { Application } from "tests/page";
 const transactions = [
   //TODO: Reactivate when fees will be stable
   //new Transaction(Account.tBTC_1, Account.tBTC_2, "0.00001", "medium"),
-  //TODO: Reactivate after blobs limit in txpool is full (-32000) resolved - JIRA: BACK-7135
-  //new Transaction(Account.tETH_1, Account.tETH_2, "0.00001", "medium"),
+  new Transaction(Account.tETH_1, Account.tETH_2.address, "0.00001", "medium"),
```

Comment:
does it make sense to test two Ethereum testnet currencies? Seems like an overkill to me

Reply:
Yes, your right, going to remove one of them